caas/kubernetes/provider/provider.go

```diff
@@ -30,6 +31,13 @@ type kubernetesEnvironProvider struct {
 	brokerGetter       func(environs.OpenParams) (caas.ClusterMetadataChecker, error)
 }
 
+const (
+	// DefaultNamespace is the default namespace to use for brokers that need to
+	// be constructed where the target namespace is not yet known. Think
+	// bootstrapping or making a broker for an unknown controller namespace.
+	DefaultNamespace = "juju"
```

Comment:
thanks for comment :-)
let's not export though unless absolutely needed outside package

Reply:
Removed because of comment below